internal/websocket/options.go

```diff
@@ -54,7 +68,10 @@ func FetchURLTimeout(d time.Duration) Option {
 		func(ws *Websocket) error {
 			if d < 0 {
 				return fmt.Errorf("%w: negative FetchURLTimeout", ErrMisconfiguredWS)
+			} else if d == 0 {
+				d = fetchUrlTimeoutDefault
```

Comment:
@schmidtw I added defaults in case the agent starts with only a `fetch_url` and nothing else

```yaml
client:
  fetch_url: https://localhost:8080/api/v2/device
```